### PR TITLE
Defect/cd 118 missing chrome storage local

### PR DESCRIPTION
**Pull request by duc-q**
### Summary
There was errors while web app tried to fetch connected account from Chrome storage when it shouldn't use this API. 
This updates get/set chrome storage API for extension only, otherwise fallback to regular local storage. 

### Checklist (If you won't pass this checklist please comment why)

-   [X ] I removed all commented or unnecessary code.

### CODE REVIEW

**Comment by duc-q** [comment]
Be careful when working with setting/getting user info from local storage.
- Web app will use local storage (Chrome storage doesn't support regular web app)
- Extension will use `chrome.storage.local`

Root cause:
`<Heading />` from web-app calls `initConnectedAccountFromLocalStorage`, which fetch data from storage. When migrating from localStorage to chrome API, I accidentally updated without awaring that this function was used only in Heading.

The error log was from `catch` handler, that's good one, which indicates some errors for developers tracing purpose.

File: client/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
 		"formik": "^2.2.9",
 		"fuse.js": "^6.5.3",
 		"identicon.js": "^2.3.3",
+		"jest-chrome": "^0.7.2",
```
Comment (gaupoit) [inline client/package.json:32]: Should we move it to `devDependencies`?

Comment (duc-q) [reply client/package.json:32]: correct, updating now

**Comment by gaupoit** [comment]
> Be careful when working with setting/getting user info from local storage.
> 
> * Web app will use local storage (Chrome storage doesn't support regular web app)
> * Extension will use `chrome.storage.local`
> 
> Root cause: `<Heading />` from web-app calls `initConnectedAccountFromLocalStorage`, which fetch data from storage. When migrating from localStorage to chrome API, I accidentally updated without awaring that this function was used only in Heading.
> 
> The error log was from `catch` handler, that's good one, which indicates some errors for developers tracing purpose.

The bug fixed on my side. Please update related UTs. Cheers.

**Review by djhahe — APPROVED** [review]

File: client/src/actions/userActions.utils.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
-import { setChromeStorageLocal, getChromeStorageLocal } from '@cd/services/localStorage';
+import { CONNECTED_ACCOUNT_STORAGE_PATH } from '@cd/constants/settings';
+import { isUsingExtension, setLocalStorageValue, setChromeStorageLocal, getChromeStorageLocal } from '@cd/services/localStorage';
 
 const getConnectedAccountChromeLocalStorage = async () => {
-  return getChromeStorageLocal(["publicKey", "loginOptions"]);
+  try {
+    return getChromeStorageLocal(["publicKey", "loginOptions"]);
+  } catch (err) {
+    return undefined;
```
Comment (gaupoit) [inline client/src/actions/userActions.utils.js:8]: Seem you missed the throw exception case @duc-q .

Screen Shot 2022-08-14 at 11 43 46

I will merge this PR first for, we add this case later in another PR. thanks.

**Pull request merged**